### PR TITLE
ansible-scylla-node: Allow the user to use a same certificate for all the nodes

### PR DESCRIPTION
**Pull request by igorribeiroduarte**

### CODE REVIEW

**Comment by tarzanek** [comment]
so with wildcard certificate support I guess https://github.com/scylladb/scylla-ansible-roles/issues/151 could be closed too

File: ansible-scylla-node/tasks/ssl.yml [hunk 1]
```diff
@@ -1,18 +1,20 @@
 ---
+
 - set_fact:
-    _localhost_cert_path: "{{ scylla_ssl.localhost_cert_path | default(inventory_dir) }}"
+    _localhost_cert_path: "{{ scylla_ssl.localhost_cert_path | default(inventory_dir + '/ssl/' + inventory_hostname + '/' + inventory_hostname + '.crt') }}"
+    _localhost_cert_key_path: "{{ scylla_ssl.localhost_cert_key_path | default(inventory_dir + '/ssl/' + inventory_hostname + '/' + inventory_hostname + '.pem') }}"
```
Comment (tarzanek) [inline ansible-scylla-node/tasks/ssl.yml:5]: why do you define them above if they get overriden anyways?

**Comment by igorribeiroduarte** [comment]
> so with wildcard certificate support I guess #151 could be closed too

The issue above is still relevant when we are using certificates generated by this role, but it's out of the scope of the current PR

File: ansible-scylla-node/tasks/ssl.yml [hunk 2]
```diff
@@ -35,11 +37,11 @@
   block:
     - name: Check if CA crt file exists
       stat:
-        path: "{{ _localhost_cert_path }}/ssl/ca/{{ scylla_cluster_name }}-ca.crt"
+        path: "{{ inventory_dir }}/ssl/ca/{{ scylla_cluster_name }}-ca.crt"
```
Comment (tarzanek) [inline ansible-scylla-node/tasks/ssl.yml:40]: I guess you wanted a way to provide a custom dir and not to hardcode it ...

Comment (tarzanek) [reply ansible-scylla-node/tasks/ssl.yml:40]: or wait, just CA has to be in same directory as inventory
and node certs can be elsewhere ?

**Review by tarzanek — APPROVED** [review]

**Review by tarzanek — APPROVED** [review]
thnx for the truststore resolution!

**Pull request merged**